x-pack/plugins/observability/public/components/app/section/apm/index.tsx

```diff
@@ -21,29 +21,34 @@ import { StyledStat } from '../../styled_stat';
 import { onBrushEnd } from '../helper';
 
 interface Props {
-  startTime?: string;
-  endTime?: string;
+  absoluteTime: { start?: number; end?: number };
+  relativeTime: { start: string; end: string };
```

Comment:
I don't think these should be optional.
```suggestion
  absoluteTime: { start: number; end: number };
  relativeTime: { start: string; end: string };
```

Reply:
Explanation why https://github.com/elastic/kibana/pull/71259#discussion_r452744232